### PR TITLE
Onboarding: Launch task UI fixes

### PR DESCRIPTION
**Pull request by 0nko**
This PR fixes 2 issues:

1. Onboarding tasks were not ordered correctly
2. The UI of the launch task preview (unreadable trial banner color in dark mode and preview padding)

| | | |
|---|---|---|
| image | image | image |

### CODE REVIEW

**Comment by JorgeMucientes** [comment]
Thanks for working on this @0nko ! The code and changes look good. However, I have some doubts about the change on the webview preview when the store is not launched. I might be unaware of more recent changes, but afaik the webview preview before launching should be full screen and not in "card" mode: VyLr7LvKodHE4kINfBE7Lw-fi-1644-82386

Screenshot 2023-03-29 at 15 05 43

I don't have a strong opinion on the small design change, I actually think it looks good. But I think is we should be aligned with iOS (cc: @selanthiraiyan) on that as well as keep @annchichi updated with the possible change before merging.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/profiler/StoreProfilerScreen.kt [hunk 1]
```diff
@@ -292,7 +292,7 @@ fun CategoriesContentPreview() {
                     StoreProfilerOptionUi(
                         name = "Electronics and Software",
                         key = "",
-                        isSelected = false
+                        isSelected = true
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/profiler/StoreProfilerScreen.kt:295]: 👍🏼

**Comment by 0nko** [comment]
Thanks, @JorgeMucientes! 

> I might be unaware of more recent changes, but afaik the webview preview before launching should be full screen and not in "card" mode

The original implementation was full screen but it had rounded corners and I noticed you partially copied it from the Installation screen, so I assumed you it was the intention to have the same preview design which got unintentionally broken. Anyway, let's see what @annchichi says.

**Comment by annchichi** [comment]
Heya, thanks for the ping. If it's a preview, we should have the user view in full screen, and when they can launch it, we have the second screen, which has rounded corners with a frame. :) 
CleanShot 2023-03-30 at 21 02 22@2x

**Comment by 0nko** [comment]
Thanks for the review, @JorgeMucientes, @annchichi. I've made the change and the non-launched preview is full-screen now.

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
Thank for applying the changes @0nko 🙏🏼 

Code looks good and works well. Let's :shipit: 

I've updated this branch with `trunk` because I think that will resolve the strings Lint issues.

**Comment by 0nko** [comment]
@JorgeMucientes Thanks!

**Pull request merged**